canonical-web-and-design/ubuntu.com
# Update edge month

## Done

- Updated existing edge month takeover with [new copy](https://docs.google.com/spreadsheets/d/1kunh8_7QioTrRwKR5Of9foxMTR1m_o_JCMrBgHgt-0A/edit#gid=2113339190), and used new takeover template
- Updated the order of webinars on /engage/edge-month according to [Alex's comments/brief](https://github.com/canonical-web-and-design/web-squad/issues/1742#issue-498888168)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Refresh the page until you see a takeover with the title "An intro to edge computing"
- See that the copy matches the [brief](https://docs.google.com/spreadsheets/d/1kunh8_7QioTrRwKR5Of9foxMTR1m_o_JCMrBgHgt-0A/edit#gid=2113339190). Bask in how pretty it looks with the new suru.
- Click the CTA, see that the webinars on the page are in the following order:
  - A guide to edge computing
  - Using kubernetes at the edge
  - Using MAAS at the edge
  - CTOs guide to the telco edge

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1740 & https://github.com/canonical-web-and-design/web-squad/issues/1741
